pkg/tests/service/errors.go

```diff
@@ -19,6 +19,7 @@ import "fmt"
 var (
 	ErrServiceNotExist     = fmt.Errorf("service not exist")
 	ErrServiceNotStarted   = fmt.Errorf("service not started")
+	ErrStoreNotReported    = fmt.Errorf("store not reported")
 	ErrInvalidServiceIndex = fmt.Errorf("invalid service index")
 	ErrFailAllocatePort    = fmt.Errorf("fail to allocate port")
```

Comment:
why use fmt.Errorf ?  There has no any parameter

Reply:
Be used to using `fmt.Errorf`, just personal habit.
Do I need to replace `fmt.Errorf` with `errors.New`?